res/skins/Tango/button_looprollsize.xml

```diff
@@ -0,0 +1,17 @@
+<!--
```

Comment:
If you forget to `git add` a new file before committing, if you have not made any other commits yet, it is easy to add it to the last commit. `git add` the new file, then `git commit --amend`. You'll have to use the `--force` option with `git push` if you already pushed the incomplete commit. That is fine as long as no one else is directly working from your branch (if they are, and you force push, that creates a messy situation for them).